src/snowcli/cli/snowpark/procedure_coverage/report.py

```diff
@@ -9,16 +9,11 @@
 
 from snowcli import config, utils
 from snowcli.config import AppConfig
-from snowcli.utils import conf_callback, generate_deploy_stage_name
+from snowcli.utils import check_for_connection, generate_deploy_stage_name
 
 from . import app
+from ...sql import ConnectionOption
```

Comment:
It doesn't look like the best way for this import.

Reply:
Good catch